content/docs/gke/gcp-e2e.md

```diff
@@ -736,26 +737,34 @@ The example comes with a simple web front end that can be used with your model.
     ...
     spec:
     containers:
-    - image: gcr.io/<your-project>/<your-deployment-name>
+    - image: gcr.io/<your-project>/<your-deployment-name>-web-ui
     ...
     ```
 
     (You can choose to use the image already deployed to Container Registry. In
     that case, you do not need to edit the `deployment.yaml` file.)
 
 3. Deploy the web front end to your cluster:
-    
+
     ```
     kustomize build . |kubectl apply -f -
     ```
 
-    Now there should be a new web UI running in the cluster. You can see the 
-    **web-ui** entry on the [GKE Workloads page][gcp-console-workloads] and on 
+    Now there should be a new web UI running in the cluster. You can see the
+    **web-ui** entry on the [GKE Workloads page][gcp-console-workloads] and on
     the [Services page][gcp-console-services].
 
 ### Access the web UI in your browser
 
-Follow these steps to access the web UI in your web browser. It may take a few 
+The web-ui service added is of type `ClusterIP`, meaning it can’t be accessed from outside the cluster. In order to load the web UI in your web browser, you have to set up a direct connection to the cluster:
+
+```sh
+kubectl port-forward svc/web-ui 8080:80
+```
+
+Then, visit `http://localhost:8080/` to access the web UI in your browser. (If you're working from the [Cloud Shell](https://cloud.google.com/shell/), you can instead click the ['web preview'](https://cloud.google.com/shell/docs/using-web-preview) button and then select "Preview on Port 8080" to connect.)
+
+<!-- Follow these steps to access the web UI in your web browser. It may take a few
```

Comment:
As you suggested ;) I'd recommend removing this content rather than commenting it out. The commented text is visible in the source preview, since it's HTML, and it's not necessary here. We can retrieve it from version control if we need it.

QQ: I know we had problems with port-forwarding in the distant past. Does it work reliably now?

Reply:
good question -- it works in this case b/c we're connecting directly to the 'web-ui' service, not going through the KF dashboard.  AFAIK there are still issues with port-forwarding to dashboard, especially w/ IAP endpoint.

Reply:
Thanks!